extensions/gdx-setup/src/com/badlogic/gdx/setup/DependencyBank.java

```diff
@@ -42,14 +42,14 @@
 	//Project plugins
 	static String gwtPluginImport = "org.wisepersist:gwt-gradle-plugin:1.0.13";
 	static String grettyPluginImport = "org.gretty:gretty:3.0.2";
-	static String androidPluginImport = "com.android.tools.build:gradle:3.4.1";
+	static String androidPluginImport = "com.android.tools.build:gradle:3.4.3";
 	static String roboVMPluginImport = "com.mobidevelop.robovm:robovm-gradle-plugin:" + roboVMVersion;
 
 
 	//Extension versions
 	static String box2DLightsVersion = "1.4";
 	static String ashleyVersion = "1.7.0";
-	static String aiVersion = "1.8.0";	
+	static String aiVersion = "1.8.2";	
```

Comment:
As this has nothing to do with Gradle, this change should be mentioned in PR title, too.

Reply:
I was just wondering why gdx-ai is stuck on a 3-or-more-year-old version. It's supposed to be a minor update, anyway, and I always use 1.8.2 in my code that uses gdx-ai.